src/OSPSuite.UI/Properties/licenses.licx

```diff
@@ -0,0 +1,4 @@
+DevExpress.XtraEditors.ImageComboBoxEdit, DevExpress.XtraEditors.v21.2, Version=21.2.3.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a
```

Comment:
REmove this file..please :)

Reply:
done. Should we somehow add this to gitignore?